outputs/costs_2020.csv

```diff
@@ -1,18 +1,83 @@
+Ammonia cracker,lifetime,30.0,years,-,
+CCGT,FOM,3.329545,%/year,"Danish Energy Agency, technology_data_for_el_and_dh_-_0009.xlsx","05 Gas turb. CC, steam extract.:  Fixed O&M"
 CCGT,VOM,4.4,EUR/MWh,"Danish Energy Agency, technology_data_for_el_and_dh_-_0009.xlsx","05 Gas turb. CC, steam extract.:  Variable O&M"
 CCGT,c_b,1.8,50oC/100oC,"Danish Energy Agency, technology_data_for_el_and_dh_-_0009.xlsx","05 Gas turb. CC, steam extract.:  Cb coefficient"
 CCGT,c_v,0.15,50oC/100oC,"Danish Energy Agency, technology_data_for_el_and_dh_-_0009.xlsx","05 Gas turb. CC, steam extract.:  Cv coefficient"
 CCGT,efficiency,0.56,per unit,"Danish Energy Agency, technology_data_for_el_and_dh_-_0009.xlsx","05 Gas turb. CC, steam extract.:  Electricity efficiency, annual average"
 CCGT,investment,880.0,EUR/kW,"Danish Energy Agency, technology_data_for_el_and_dh_-_0009.xlsx","05 Gas turb. CC, steam extract.:  Nominal investment"
 CCGT,lifetime,25.0,years,"Danish Energy Agency, technology_data_for_el_and_dh_-_0009.xlsx","05 Gas turb. CC, steam extract.:  Technical lifetime"
+CH4 (g) fill compressor station,FOM,1.7,%/year,Assume same as for H2 (g) fill compressor station.,
+CH4 (g) fill compressor station,investment,1498.948261,EUR/MW_CH4,"Guesstimate, based on H2 (g) pipeline and fill compressor station cost.",
+CH4 (g) fill compressor station,lifetime,20.0,years,Assume same as for H2 (g) fill compressor station.,
+CH4 (g) pipeline,FOM,1.5,%/year,Assume same as for H2 (g) pipeline in 2050 (CH4 pipeline as mature technology).,
+CH4 (g) pipeline,investment,78.997841,EUR/MW/km,Guesstimate.,
+CH4 (g) pipeline,lifetime,50.0,years,Assume same as for H2 (g) pipeline in 2050 (CH4 pipeline as mature technology).,
+CH4 (g) submarine pipeline,FOM,3.0,%/year,"d’Amore-Domenech et al (2021): 10.1016/j.apenergy.2021.116625 , supplementary material.",
+CH4 (g) submarine pipeline,investment,114.8928,EUR/MW/km,Kaiser (2017): 10.1016/j.marpol.2017.05.003 .,
+CH4 (g) submarine pipeline,lifetime,30.0,years,"d’Amore-Domenech et al (2021): 10.1016/j.apenergy.2021.116625 , supplementary material.",
+CH4 (l) transport ship,FOM,3.5,%/year,"Fasihi et al 2017, table 1, https://www.mdpi.com/2071-1050/9/2/306",
+CH4 (l) transport ship,capacity,58300.0,t_CH4,"Calculated, based on Fasihi et al 2017, table 1, https://www.mdpi.com/2071-1050/9/2/306",
+CH4 (l) transport ship,investment,151000000.0,EUR,"Fasihi et al 2017, table 1, https://www.mdpi.com/2071-1050/9/2/306",
+CH4 (l) transport ship,lifetime,25.0,years,"Fasihi et al 2017, table 1, https://www.mdpi.com/2071-1050/9/2/306",
+CH4 evaporation,FOM,3.5,%/year,"Fasihi et al 2017, table 1, https://www.mdpi.com/2071-1050/9/2/306",
+CH4 evaporation,investment,0.277628,"EUR/kW_CH4; and
+EUR/MW_CH4","Calculated, based on Fasihi et al 2017, table 1, https://www.mdpi.com/2071-1050/9/2/306",
```

Comment:
This is an odd line break and wrong with two disagreeing units.

Reply:
The new feature which allows to specify new technologies via the external `manual_input.csv` file takes all distinct units (and sources, further descriptions, ...) of the same technology and combines them into one field, delimited by "; and" .

--> This is caused by a malformed input in `manual_input.csv` where for the technology the investment costs are specified in EUR/kW_CH4 for 2030 and in EUR/MW_CH4 in 2040. I'll push a fix.

It only affects CH4 evaporation and CH4 liquefaction, correct?